Errors that occurr when stripping VCS are being ignored

I am running into an issue on Windows where attempting to delete a file inside the VCS folder results in an error. The error is currently being ignored, and resulting in the stripping operation to fail silently.

After adding error logging, I am observing the following:
```
C:\code\Go\src\github.com\alexbrand\test>glide install --strip-vcs
[INFO]  Downloading dependencies. Please wait...
[INFO]  --> Fetching updates for github.com/spf13/cobra.
[INFO]  Setting references.
[INFO]  --> Setting version for github.com/spf13/cobra to 6a8bd97bdb1fc0d08a83459940498ea49d3e8c93.
[INFO]  Removing version control data from vendor directory...
[INFO]  Removing: vendor\github.com\spf13\cobra\.git
[ERROR] remove vendor\github.com\spf13\cobra\.git\objects\pack\pack-97ef767fc6fcc6f9040ae32b4cecabd76bd1761d.idx: Access is denied.
An Error has occurred
```

--- a/action/get.go
+++ b/action/get.go
@@ -95,7 +95,10 @@ func Get(names []string, installer *repo.Installer, insecure, skipRecursive, str
 
 	if strip {
 		msg.Info("Removing version control data from vendor directory...")
-		gpath.StripVcs()
+		err := gpath.StripVcs()
+		if err != nil {
+			msg.Err("Unable to strip version control data: %s", err)
+		}
 	}
 
 	if stripVendor {

--- a/action/install.go
+++ b/action/install.go
@@ -72,7 +72,10 @@ func Install(installer *repo.Installer, strip, stripVendor bool) {
 
 	if strip {
 		msg.Info("Removing version control data from vendor directory...")
-		gpath.StripVcs()
+		err := gpath.StripVcs()
+		if err != nil {
+			msg.Err("Unable to strip version control data: %s", err)
+		}
 	}
 
 	if stripVendor {

--- a/action/update.go
+++ b/action/update.go
@@ -117,7 +117,10 @@ func Update(installer *repo.Installer, skipRecursive, strip, stripVendor bool) {
 
 	if strip {
 		msg.Info("Removing version control data from vendor directory...")
-		gpath.StripVcs()
+		err := gpath.StripVcs()
+		if err != nil {
+			msg.Err("Unable to strip version control data: %s", err)
+		}
 	}
 
 	if stripVendor {

